test/functional/plugin/tutor_spec.lua

```diff
@@ -0,0 +1,25 @@
+local helpers = require('test.functional.helpers')(after_each)
+local command = helpers.command
+local feed = helpers.feed
+local is_os = helpers.is_os
+local skip = helpers.skip
+local eval = helpers.eval
+local eq = helpers.eq
+local clear = helpers.clear
+
+describe(':Tutor', function()
+  clear({ args = { '-u', 'NORC' } })
+  command("Tutor")
+
+  it('should apply transformation for Unix', function()
+    skip(is_os('win'), 'not on Unix')
+    feed("/!ls<CR>")
+    eq(true, eval('line(".")') > 1)
+  end)
+
+  it('should apply transformation for Windows', function()
+    skip(not is_os('win'), 'not on Windows')
+    feed("/!dir<CR>")
+    eq(true, eval('line(".")') > 1)
+  end)
```

Comment:
just make this one test with a `if` check.

a screen test is probably better, and adds a ton of coverage with a single test.  See [test/functional/example_spec.lua](https://github.com/neovim/neovim/blob/master/test/functional/example_spec.lua) for example. You can [run the test](https://github.com/neovim/neovim/blob/master/test/README.md#running-tests) like this:

    TEST_FILE=test/functional/plugin/tutor_spec.lua make functionaltest

Reply:
I didn't find a way to check the line number after the jump through the `screen` so I checked the whole line content.